lib/Steps/Integrations/Cordova.ts

```diff
@@ -190,31 +190,68 @@ export class Cordova extends BaseIntegration {
         shellPath: '/bin/sh',
         shellScript:
           // eslint-disable-next-line prefer-template
+          '#!/bin/bash\\n' +
+          '# print commands before executing them and stop on first error\\n' +
+          'set -x -e\\n' +
           'echo "warning: uploading debug symbols - set SENTRY_SKIP_DSYM_UPLOAD=true to skip this"\\n' +
-          'if [ -n "$SENTRY_SKIP_DSYM_UPLOAD" ]; then\\n' +
-          '  echo "warning: skipping debug symbol upload"\\n' +
-          '  exit 0\\n' +
-          'fi\\n' +
-          'export SENTRY_PROPERTIES=' +
-          cwd +
-          '\\n' +
-          'function getProperty {\\n' +
-          '    PROP_KEY=$1\\n' +
-          '    PROP_VALUE=`cat $SENTRY_PROPERTIES | grep "$PROP_KEY" | cut -d\'=\' -f2`\\n' +
-          '    if [ -z "$PROP_VALUE" ]; then\\n' +
-          '        echo "plugins/sentry-cordova/node_modules/@sentry/cli/bin/sentry-cli"\\n' +
-          '    else\\n' +
-          '        echo $PROP_VALUE\\n' +
+          '[ -z "$SENTRY_FORCE_FOREGROUND"] && SENTRY_FORCE_FOREGROUND=true\\n' +
+          '[[ "$SENTRY_FORCE_FOREGROUND" == true ]] && SENTRY_FORCE_FOREGROUND_FLAG="--force-foreground"\\n' +
+          'get_node_command() {\\n' +
+          '  if [ -x "$(command -v ${NODE_BINARY})" ]; then\\n' +
+          '    echo "${NODE_BINARY}"\\n' +
+          '    return 0\\n' +
+          '  fi\\n' +
+          '  if [ -x "$(which node)" ]; then\\n' +
+          '    echo "node"\\n' +
+          '    return 0\\n' +
+          '  fi\\n' +
+          '  NVM_NODE_VERSIONS_DIR="$HOME/.nvm/versions/node"\\n' +
+          '  if [ -d "$NVM_NODE_VERSIONS_DIR" ] && [ "$(ls -A $NVM_NODE_VERSIONS_DIR)" ]; then\\n' +
+          '    HIGHEST_VERSION=$(ls -v "$NVM_NODE_VERSIONS_DIR" | tail -n 1)\\n' +
+          '    NODE_BINARY="$NVM_NODE_VERSIONS_DIR/$HIGHEST_VERSION/bin/node"\\n' +
+          '    if [ -x "$NODE_BINARY" ]; then\\n' +
+          '      echo "$NODE_BINARY"\\n' +
+          '      return 0\\n' +
           '    fi\\n' +
+          '  fi\\n' +
+          '  echo ""\\n' +
+          '  return 0\\n' +
           '}\\n' +
```

Comment:
Are we getting complains about the node not being set?
I think we should just allow to use `NODE_BINARY` if global node is not set.
We can make sure that the `NODE_BINARY` by the wizard.

For example RN [deprecated](https://github.com/facebook/react-native/blob/c684f9fc62e865a5ecbd8ade0f5628f016eebcb2/packages/react-native/scripts/xcode/with-environment.sh#L41) the dynamic node lookup

Reply:
I experienced it where node wasn't found by Xcode while it was found by the terminal